src/runtime/mpz.cpp

```diff
@@ -757,6 +763,10 @@ mpz & mpz::operator%=(mpz const & o) {
     return rem(o.m_size, o.m_digits);
 }
 
+mpz divexact(mpz const & n, mpz const & d) {
```

Comment:
This doesn't link without
```suggestion
mpz mpz::divexact(mpz const & n, mpz const & d) {
```
#9106

Reply:
Yeah, my bad, thanks for fixing